edx_sysadmin/templates/edx_sysadmin/users.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'edx_sysadmin/base.html' %}
+
+{% load i18n static %}
+
+{% block panel %}
+    {% if disclaimer %}
+		<div class="disclaimer">
+            <b>Disclaimer</b>: Account Registration API is not function in your platform, User Account will be created through DB Models
```

Comment:
I think this disclaimer should explain which feature flags should be turned on to use registration API.

Reply:
Updated.
![image](https://user-images.githubusercontent.com/42243411/112264861-50496300-8c93-11eb-8f24-0222a3fd2a4c.png)